vars/Windows.yml

```diff
@@ -0,0 +1,17 @@
+---
+# File: Windows.yml - Windows OS variables for Consul
+
+# paths
+consul_windows_path: /ProgramData/consul
```

Comment:
Fun fact: forward slashes now work almost as well as backslashes do in Windows! Folder structure is still different though.